galaxy_ng/app/settings.py

```diff
@@ -30,6 +30,10 @@
 # created by the initial-data.json data fixture
 GALAXY_API_DEFAULT_DISTRIBUTION_BASE_PATH = "automation-hub"
 
+# The format for the name of the per namespace inbound repository,
+# and the associated distribution, and distribution base_path
+GALAXY_API_INBOUND_REPO_NAME_FORMAT = "inbound-{namespace_name}"
```

Comment:
I think it is better to be a constant, rather than settings.

Reply:
PR #289 has a proposed `GALAXY_API_SYNCLIST_NAME_FORMAT = "{account_number}-synclist"` in settings, im ok either way but probably should be consistent

Reply:
synclist and account_number is kind of inherently automation-hub / cloud.redhat.com specific, so lends itself to settings. And if any future versions of standalone would support synclists, the default synclist naming would need to change, so also more suitable for settings.

In theory, namespace probably won't change for c.r.c or standalone. Although I could see a case for standalone where folks deploying standalone may want to change the default inbound repos. For ex, maybe having a single inbound repo, or multiple inbound repos one for each team, or even one inbound repo for each collection. Or not having any local namespaces at all and having inbound repos.  But maybe that means inbound-repo-per-namespace is really a c.r.c specific feature which suggests driving it's behavior from settings make sense.

Reply:
Given some of those scenarios such as multiple inbound repos cannot be handled with a settings change, and since this probably won't change, I moved this to a local module constant.